scripts/commands.js

```diff
@@ -77,8 +77,7 @@ program
   .option('--enable_brave_update', 'enable brave update')
   .option('--channel <target_chanel>', 'target channel to start', /^(beta|dev|nightly|release)$/i, 'release')
   .option('--official_build <official_build>', 'force official build settings')
-  .option('--rewards_env [server]', 'switch between staging and production', /^(stag|prod)$/i)
-  .option('--rewards_reconcile_interval [reconcile_interval]', 'set reconcile interval for contribution in minutes', parseInt)
+  .option('--rewards [options]', 'options for rewards')
```

Comment:
Please document the options strings here or reference somewhere where it is documented.

Reply:
done